packages/esm-patient-registration-app/src/patient-registration/field/obs/obs-field.component.tsx

```diff
@@ -57,12 +57,13 @@ export function ObsField({ fieldDefinition }: ObsFieldProps) {
           answerConceptSetUuid={fieldDefinition.answerConceptSetUuid}
           label={fieldDefinition.label}
           required={fieldDefinition.validation.required}
+          customConceptAnswers={fieldDefinition.customConceptAnswers}
         />
       );
     default:
       return (
         <InlineNotification kind="error" title="Error">
-          Concept has unknown datatype "{concept.datatype.display}"
+          Concept for obs field "{fieldDefinition.id}" has unknown datatype "{concept.datatype.display}"
```

Comment:
Hi @brandones !
This text needs to be translated. Can you please add the translation?
Thanks!

Reply:
Do we translate error messages? @ibacher @denniskigen

Reply:
We don't translate error messages on the console, but if it's displayed to the end-user, then yes, yes we do. (Since this is an inline notification I assume it's displayed). Granted, since the text here was just tweaked we can delay that to another PR.

Reply:
Every error that gets thrown shows up to the end-user. We don't have the norm of translating those. These are like that: hopefully, only an admin should ever see it, for it indicates that something is very badly wrong. The only difference with a thrown error is that a thrown error generally crashes down to the error boundary, while this I have considered "recoverable."

Reply:
(obviously my point is not to whinge about having to add translations, but to ensure we're clear on our translation policies :stuck_out_tongue_closed_eyes: )

Reply:
You're right. The rule here is somewhat narrower, since not every error, ends up translated (though, frankly, backend error messages are also translatable, so those that aren't can be treated like bugs if they are displayed to users). 

I _do_ think that, while we haven't necessarily been consistent about it, it is good to translate these sort of error messages even if they are intended for administrators rather than end-users. We're trying to move towards a world were don't assume all administrators of the system are Anglophones (or have an English-speaking support team), which is necessary for global reach.